rover.ts

```diff
@@ -0,0 +1,27 @@
+import { Directions } from "./contants";
+import explore from "./explore";
+import Plateau from "./plateau";
+
+export default class Rover {
+    private config: object;
+    constructor(config: object) {
+        this.config = config;
+    }
+
+    // dummy method for demonstration
+    setup(){
+        // uses config
+        return `Setup successfull`
```

Comment:
If setup isn't doing anything, do we need it?

Reply:
Not really, just exists for demo (as an example for rover class with extra methods than just move).